main/guides/getting-started/sell-concert-tickets-contract-explainer.md

```diff
@@ -3,9 +3,16 @@ This smart contract is designed to mint and sell event tickets as non-fungible t
 - Tickets near the front are the most expensive
 - Tickets in the middle rows are priced between expensive and cheap seats
 - Tickets in the back are the lowest priced
+> Note: This contract simulates traditional trading i.e. between a vendor and a consumer.
+
+## Objective
+The objective of this tutorial is to teach you the following:
```

Comment:
*teach you* sounds a bit demeaning - maybe change it something like: *This tutorial has following objectives: *

Reply:
Fixed it.